services/github/github-downloads.service.js

```diff
@@ -22,7 +22,7 @@ const releaseArraySchema = Joi.alternatives().try(
   Joi.array().length(0)
 )
 
-const keywords = ['github download']
+const keywords = ['download']
```

Comment:
this is what i originally went with, but at least in my local env I was hoping to search with `github download` to narrow the results down all the way

Reply:
Oh, sorry! Did that work? If so I’ll revert it. I thought the query got split on spaces and then matched against all the keywords, so didn’t think spaces in the keywords made any difference.

Reply:
Maybe we should have both terms? Right now in the review app I can search on `download` which gives a better result than prod today, but if I search for `github d..` I get no results

Reply:
I feel like searching for "github download" should do the expected thing and we should fix that in a broader way. I've dug in a bit and it seems to be a bit involved, so I'll leave what you did for now.